client/js/components/GameView/GameOverlays.vue

```diff
@@ -104,6 +113,26 @@ export default {
       }
       return mayCounter;
     },
+    showWaitingForOpponentToDiscardJackFromDeck() {
+      return (
+        this.resolvingSeven &&
+        this.topTwoCardsCantBePlayed &&
+        (this.opponentPointTotal === 0 || this.opponentQueenCount > 0) &&
+        this.isPlayersTurn === false
```

Comment:
Using `waitingForOpponentToPlayFromDeck` above should alleviate the need for the turn check b/c we only set `waitingForOpponentToPlayFromDeck` when it is the opponent's turn
```suggestion
        (this.opponentPointTotal === 0 || this.opponentQueenCount > 0)
```